front50-core/src/main/java/com/netflix/spinnaker/front50/jackson/mixins/PipelineMixins.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.front50.jackson.mixins;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.Map;
+
+@JsonIgnoreProperties(
```

Comment:
If you look at the deleted Pipeline file in front50-core you will see the `JsonIgnore` for these properties: https://github.com/spinnaker/front50/pull/1035/files/ef8c0e5ee201defe49c325902162cfdc52e4320e#r647609879

Reply:
Hmm ... interesting. I wonder if they were still being serialized because of the backing map. 

Either way, assuming they're still being serialized it seems alright to me.